sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -8,13 +8,27 @@ import { AzureKeyCredential } from '@azure/core-auth';
 import { CommonClientOptions } from '@azure/core-client';
 import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
 export interface ActionCommon {
     disableServiceLogs?: boolean;
 }
 
+// @public
+export type ActionCustom = ActionCommon & {
```

Comment:
I think you probably want

```typescript
interface ActionCustom extends ActionCommon { ... }
```

If possible. Docs systems handle this better than the alias to intersection.

Reply:
I agree. For context, these things are auto-generated and I didn't want to duplicate them. Perhaps I should talk to the code gen crew about this but not sure about when should it use interface extension vs intersection.